conf/messages.es

```diff
@@ -246,6 +246,11 @@ results.legend.high = Alto
 results.legend.low = Bajo
 results.legend.body = Los barrios grises carecen de datos suficientes para evaluar los problemas de las banquetas. Haz clic en cualquier barrio para empezar a contribuir. Los resultados son preliminares y representan contribuciones voluntarias que aún no han sido validadas. Estamos trabajando en ello. ¡Manténte al tanto!
 
+dashboard.no.org = Sin Organización
+dashboard.edit.org = Editar
+dashboard.edit.info = Si no ve su organización, envíe un correo electrónico a <a href="mailto:sidewalk@cs.uw.edu">sidewalk@cs.uw.edu</a> con el nombre y la descripción de su organización.
```

Comment:
Could you make the email address link open in a new window? I think you just need to add `target="blank"` to the `<a>`.